Kevin Lin Headstorm Interview code/BackEnd/server.js

```diff
@@ -0,0 +1,9 @@
+const http = require('http');
+const app = require('./app');
+
+const post = process.env.PORT || 3000;
+
+const server = http.createServer(app);
+
+server.listen(port);
```

Comment:
I notices you have another server listener above:
```
const port = process.env.PORT || 3000;
app.listen(port, () => console.log('Listening on port ${port}...'));
```

Do you have two separate servers in this codebase?